### ISSUE TITLE
Low Priority - WCAG Level AAA - Link purpose - ‘Clear all’ link was ambiguous for screen reader users

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#167]
The ‘Clear all’ link was ambiguous for screen reader users out of context.

WCAG Reference:
2.4.9 Link Purpose - Link Only - (Level AAA)
[Understanding Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html) | [How to Meet Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-link-only)

**Issue ID: DAC_Link_purpose_01**

URL: https://beta.w3.org/membership/list/ 
Page: Current Members & Testimonials 
Journey 2, Step 1
<img width="830" alt="clear all button" src="https://user-images.githubusercontent.com/20167528/159256458-1947f625-afcf-424e-bca4-95a9e6a85a04.png">

For screen reader users navigating out of context of the page, the 'Clear all' link is ambiguous. Additional context should be provided in the links labelling to ensure that screen reader users are able to determine the purpose of the link from the link text alone. This information may be visually concealed to preserve the appearance of the page.

**Current Code Ref(s):** #main > div.u-full-width.hero.hero--listing > div > form > div > div:nth- child(4) > div > a
```
<a href="/membership/list/" class="button button--ghost">Clear all</a>
```

**Screen reader user comments:**

> “While exploring out of context with Tab, I located a link spoken as ‘Clear all’. I found this link text only partially descriptive, and would have preferred an unambiguous phrase such as ‘Clear all search filters’. Making such a change will ensure an easier navigation experience for assistive technology users.
> The issue is present with JAWS, NVDA.”

**Solution:**
To ensure that the purpose of the links is clear for users who rely on audio feedback to navigate, it is recommended to include additional information within each the link text. This information can be included by way of a visually hidden <span>.

**Example:**
```
 <a href="/membership/list/" class="button button--ghost">Clear all<span class="sr- only"> filters</span>
</a>
```

**CSS:**
```
.sr-only
  position:    absolute !important;
  overflow:    hidden   !important;
  white-space: nowrap   !important;
  width:   1px !important;
  height:  1px !important;
  margin: -1px !important;
  padding: 0   !important;
  border:  0   !important;
  clip: rect(1px, 1px, 1px, 1px) !important;
  -webkit-clip-path: inset(50%) !important;
  clip-path: inset(50%) !important;
}
```

**Comment by deniak** [comment]
It's now fixed.
